src/components/Node/Node.module.scss

```diff
@@ -28,6 +28,10 @@
     background-color: $red;
   }
 
+  &.declined {
+    background-color: $black;
```

Comment:
Taking a look at the design I don't think the background color is black. From a color picker it looks like `#404040`. @noranda is this the correct color?